### ISSUE TITLE
continuous loss function for classification

### ISSUE THREAD

**Issue by lacava** [lacava/feat#98]
rather than minimizing the zero one loss, we should minimize log loss / cross entropy for classification. this will allow us to reward decision functions with larger thresholds.

**Comment by lacava** [comment]
see #126
